### DISCUSSION TITLE
Sorting quality metrics calculations

### DISCUSSION THREAD

**Discussion by zjm199502** [petersenpeter/CellExplorer#86] [Q&A] [answered]
Hi peter,

I got a problem on cell-metrics. 

`isolationDistance` and `lRatio` seems didn't calculated automatically (with no mannual modification on the default cell_metrics settings).  Do I need perfom the calculations individually?
![f9b1aee42ee643c1db559a222810651](https://user-images.githubusercontent.com/60344825/204572449-44519868-e478-4b8b-9782-e6f806dc03e2.jpg)

Thank you very much in advance.

Jinming

**Comment by petersenpeter** [comment, marked as answer]
Hi Jingming

The sorting quality metrics are not included in the standard metrics, because I have not had time to code a solution that works across different probe layouts and sorting algorithms. If you do end up coding it, feel free to push your code to the repo.

Best
Peter

**Reply by zjm199502** [reply]
OK, although it's difficulty for me, I'll try to do it later.

**Reply by petersenpeter** [reply]
Which spike sorting software are you using?

**Reply by zjm199502** [reply]
Now I'm using Spyking Circus.

**Comment by zjm199502** [comment]
Now I'm using Spyking Circus发自我的 iPhone在 2022年11月30日，16:31，Peter C. Petersen ***@***.***> 写道：﻿
Which spike sorting software are you using?

**Comment by petersenpeter** [comment]
I have scripts for calculating the metrics for Neurosuite and for Kilosort, but I don't believe I have them for Spyking Circus, unfortunately.

**Reply by zjm199502** [reply]
Could you share the scripts for Neuosuite and Kilosort? I think it will provides some references

**Reply by petersenpeter** [reply]
Hi
You can find the quality metrics for Kilosort here by Nick Steinmetz: https://github.com/cortex-lab/sortingQuality
For Neurosuite, the code is actually already part of CellExplorer: https://github.com/petersenpeter/CellExplorer/blob/master/ProcessCellMetrics.m#L662

**Comment by zjm199502** [comment]
Thank you for sharing!
